src/k8sconfiguration/setup.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from codecs import open
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+# TODO: Confirm this is the right version number you want and it matches your
+# HISTORY.rst entry.
+VERSION = '0.1.7'
```

Comment:
Why '0.1.7' ? We only need to upgrade it when we publish it. I think we can use `0.1.0` until we release the first version.

Reply:
We have already released it for Private Preview customers, and have been releasing updated versions.  To differentiate from that, I'd like to keep it as version 0.1.7, unless there is a requirement to start at 0.1.0

Reply:
VERSION in `setup.py` and `HISTORY.rst` should align.